### PR TITLE
Added line and charPositionInLine to ParseTreeNode

### LINKED ISSUE #41: Line and Column Error in ParseTreeNode
Class ParseTreeNode has methods getSidx() and getEidx(), but these methods are not sufficient.

There should also be methods getBeginLine(), getBeginCharPositionInLine(), genEndLine(), and getEndCharPositionInLine().

This way, we can quickly locate where the node is in the source parsed.  Useful for output detailed error messages.

### PR DESCRIPTION
**Pull request by ghost**
This is a proposal to add begin line and begin char position in line to ParseTreeNode

### CODE REVIEW

File: inmemantlr-api/src/main/java/org/snt/inmemantlr/tree/ParseTreeNode.java [hunk 1]
```diff
@@ -246,13 +255,28 @@ public int getSidx() {
     }
 
     /**
-     * get ent index
+     * get end index
```
Comment (julianthome) [inline inmemantlr-api/src/main/java/org/snt/inmemantlr/tree/ParseTreeNode.java:258]: Thanks a lot for fixing this 👍🏼

**Comment by julianthome** [comment]
This is awesome @calogero81! Thank you very much for you contribution. 
LGTM 
As part of this PR, could you bump the version of inmemantlr to 1.9.0 and add your name in the `Authors and major contributors` section of the README.md file?

**Comment by ghost** [comment]
Thank you, @julianthome,  bumped version and updated README.md file.

**Comment by julianthome** [comment]
Awesome! Thanks again @calogero81 👍🏼

**Pull request merged**